static/js/components/SourceTable.jsx

```diff
@@ -443,6 +447,56 @@ const SourceTable = ({
     );
   };
 
+  const renderSpectrumExists = (dataIndex) => {
+    const source = sources[dataIndex];
+    return source.spectrum_exists ? (
+      <CheckIcon
+        size="small"
+        key={`${source.id}_spectrum_exists`}
+        color="primary"
+      />
+    ) : (
+      <ClearIcon
+        size="small"
+        key={`${source.id}_spectrum_exists`}
+        color="secondary"
+      />
+    );
+  };
+
+  const renderPeakMagnitude = (dataIndex) => {
+    const source = sources[dataIndex];
+    const peakPoint = source.photometry
+      .filter((point) => point.flux)
+      .sort((a, b) => a.flux - b.flux)[0];
+    return (
+      <Tooltip title={`${(mjdNow - peakPoint.mjd).toFixed(2)} days ago`}>
+        <div>{`${flux_to_mag(peakPoint.flux).toFixed(4)}`}</div>
+      </Tooltip>
+    );
+  };
+
+  const renderLatestMagnitude = (dataIndex) => {
+    const source = sources[dataIndex];
+    const latestPoint = source.photometry
+      .filter((point) => point.flux)
+      .sort((a, b) => b.mjd - a.mjd)[0];
+    return (
+      <Tooltip title={`${(mjdNow - latestPoint.mjd).toFixed(2)} days ago`}>
+        <div>{`${flux_to_mag(latestPoint.flux).toFixed(4)}`}</div>
+      </Tooltip>
+    );
+  };
+
+  const renderTNSName = (dataIndex) => {
+    const source = sources[dataIndex];
+    return (
+      <div>
+        {source.altdata && source.altdata.tns ? source.altdata.tns.name : ""}
```

Comment:
Is this populated currently?

Reply:
Not in our demo data at least. But this is where the source handler looks for the `has_tns_name` query parameter (https://github.com/skyportal/skyportal/blob/50adea8c2be94f7fd92da317527e471445fd33ed/skyportal/handlers/api/source.py#L561-L562), so I assumed this would be where we would find the TNS name if it existed.